src/NuGet.Core/NuGet.Protocol/ServiceTypes.cs

```diff
@@ -7,16 +7,18 @@ public static class ServiceTypes
     {
         public static readonly string Version200 = "/2.0.0";
         public static readonly string Version300beta = "/3.0.0-beta";
+        public static readonly string Version300rc = "/3.0.0-rc";
```

Comment:
If this is internal, does anything break? I'm guessing no, so we can probably do that too. No need to increase the public API despite the precedent. 
Thoughts @NuGet/nuget-client

Reply:
I agree with above comment. There is no need to add new public API surface if it is not required.

Reply:
Team Review: In this case it doesn't matter since they are readonly strings and we'll never change them anyway